themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
>  I personally like the existing padding more than 0.5 em.
I'm with you there, but unfortunately this plugin is literally just a plug-in of 3rd party code. `.hljs { ... padding: 0.5em; ... }` is all over it. See for yourself: https://github.com/isagalaev/highlight.js/search?utf8=%E2%9C%93&q=.hljs+padding%3A&type=Code